calculator-project/src/Components/ResultInputs.js

```diff
@@ -0,0 +1,19 @@
+import React, {Component} from 'react';
+import "./ResultInput.css"
+
+class ResultInputs extends Component {
+
+
+    render() {
+        let {result} = this.props;
+        return (
+            <div className="result">
+                <p>{result}</p>
```

Comment:
this is a great place for a ternary that checks for an empty string and if it is, display a 0